src/cli.py

```diff
@@ -41,18 +45,24 @@ async def cli(args, script_root):
         elif args.command == "upgrade":
             upgrade()
         elif args.command == "test":
+            cairo_paths: List[Path] = args.cairo_path or []
+            if protostar_dir:
+                cairo_paths.append(protostar_dir / "cairo")
             await run_test_runner(
                 getattr(args, "tests-root"),
                 project=current_project,
                 omit=args.omit,
                 match=args.match,
-                cairo_paths=args.cairo_path,
+                cairo_paths=cairo_paths,
             )
         elif args.command == "build":
+            cairo_paths: List[Path] = args.cairo_path
+            if protostar_dir:
```

Comment:
Will be any problem in case of duplicate? I guess not, just want to make sure.

Reply:
I don't know tbh. I changed the code to skip append if path is already included.